azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/MigrateVirtualMachine.java

```diff
@@ -23,20 +23,93 @@
  *   - Migrate the virtual machine to use managed disk.
  */
 public final class MigrateVirtualMachine {
+
     /**
-     * Main entry point.
-     * @param args the parameters
+     * Main function which runs the actual sample.
+     *
+     * @param azure instance of the azure client
+     * @return true if sample runs successfully
      */
-    public static void main(String[] args) {
-
+    public static boolean runSample(Azure azure) {
         final String linuxVmName = Utils.createRandomName("VM1");
         final String rgName = Utils.createRandomName("rgCOMV");
         final String userName = "tirekicker";
         final String password = "12NewPA$$w0rd!";
         final Region region = Region.US_WEST_CENTRAL;
 
         try {
+            //=============================================================
+            // Create a Linux VM using a PIR image with un-managed OS and data disks
+
+            System.out.println("Creating an un-managed Linux VM");
+
+            VirtualMachine linuxVM = azure.virtualMachines().define(linuxVmName)
```

Comment:
pls use indentation under defineChild() flows, until (and including) attach, to make things more readable and consistent with the rest of the samples.
for example:
```
.defineFoo()
    .withBar()
    .attach()
.defineFoo()
    .withBar()
    .attach()
.create()
```